### ISSUE TITLE
How I can use variable from JSON array? 

### ISSUE THREAD

**Issue by maximalexeev** [a85/POSTMan-Chrome-Extension#4483]
I have JSON responce, looks as: 
{
    "resourceType": "Parameters",
    "parameter": [
        {
            "name": "Order",
            "resource": {
                "resourceType": "Order",
                "id": "a0777178-7177-4b6e-8f45-2e3601080901",
                "meta": {
                    "versionId": "vc7d07f0-d679-4c65-aa62-14e9093f383b",
                    "lastUpdated": "2018-04-23T11:50:29.765+00:00"
                }, etc... 

I need "id" value for another request, but I can`t got it. 

I can only get array by 
pm.environment.set("xxx", pm.response.json().parameter);

And no way to get "id" value from array

How I can do it?

**Comment by mccannt** [comment]
@MaximA10rus 
So for example if this was your call ...
`curl -X POST \
  http://postman-echo.com/post \
  -H 'Cache-Control: no-cache' \
  -H 'Content-Type: application/json' \
  -H 'Postman-Token: b6012e04-f517-4c88-83b3-518ee597bfef' \
  -d '{
	"resourceType": "Parameters",
	"parameter": [{
		"name": "Order",
		"resource": {
			"resourceType": "Order",
			"id": "a0777178-7177-4b6e-8f45-2e3601080901",
			"meta": {
				"versionId": "vc7d07f0-d679-4c65-aa62-14e9093f383b",
				"lastUpdated": "2018-04-23T11:50:29.765+00:00"
			}
		}
	}]
}'`

Then you would want to get the data from the response something like this ...
`
var jsonData = pm.response.json();
console.log(jsonData.json.parameter[0].resource.id);
`

Basically you can put that into a variable like this ...
`
pm.environment.set("resourceId", jsonData.json.parameter[0].resource.id);
`
You just need to make sure to select the correct element in the array and you should be all set. 

Hope this helps
